test/src/test/java/hudson/slaves/JNLPLauncherTest.java

```diff
@@ -212,23 +234,20 @@ private void launchJnlpAndVerify(Computer c, ArgumentListBuilder args) throws Ex
     }
 
     /**
-     * Determines the link to the .jnlp file.
+     * Adds a JNLP {@link Slave} to the system and returns it.
      */
-    private String getJnlpLink(Computer c) throws Exception {
```

Comment:
why would you remove this method? Seems unrelated

Reply:
The view for a nested launcher is different (actually none for the unit-test launcher) and therefor it's not possible to scrape the JNLP link from the page. Also the scraping was hardly necessary, the link hasn't changed since 10 years, so I just replaced it with a hard-coded URL path.

Reply:
agreed